src/middleware-set-current-time.js

```diff
@@ -12,12 +12,20 @@ videojs.use('*', (player) => {
     // level), this middleware will capture the action. For internal seeks (generated at
     // the tech level), we use a wrapped function so that we can handle it on our own
     // (specified elsewhere).
-    setCurrentTime: (time) => {
-      if (player.vhs && player.currentSource().src === player.vhs.source_.src) {
+    setCurrentTime(time) {
+      if (player.vhs &&
+          player.currentSource().src === player.vhs.source_.src) {
         player.vhs.setCurrentTime(time);
       }
 
       return time;
+    },
+
+    play() {
+      if (player.vhs &&
+          player.currentSource().src === player.vhs.source_.src) {
+        player.vhs.setCurrentTime(player.currentTime());
```

Comment:
I know this is meant to be a short-term fix, but we should add some comments about what we're doing and why it works for the replay case.